quiz.js

```diff
@@ -37,9 +50,12 @@ db.dropDatabase();
 
 // your code begins here
 
-
-
-
+db.houses.update({ name: 'Stark' }, { $push: { people: "Ned Stark" }});
+db.houses.update({ name: 'Stark' }, { $push: { people: "Arya Stark" }});
+db.houses.update({ name: 'Stark' }, { $push: { people: "Sansa Stark" }});
+db.houses.update({ name: 'Targaryen' }, { $push: { people: "Viserys Targaryen" }});
+db.houses.update({ name: 'Targaryen' }, { $push: { people: "Daenerys Targaryen" }});
+db.houses.update({ name: 'Arryn' }, { $push: { people: "Jon Arryn" }});
 
```

Comment:
Yep, this is correct! One shorter way to write this would be to take advantage of [$each](http://docs.mongodb.org/manual/reference/operator/update/each/), like so:
```
db.houses.update({name: 'House Stark'}, { $push: { people: { $each: ['Ned Stark', 'Arya Stark', 'Sansa Stark']}}});
 
db.houses.update({name: 'House Targaryen'}, { $push: { people: { $each: ['Viserys Targaryen', 'Daenerys Targaryen']}}});

db.houses.update({name: 'House Arryn'}, { $push: { people: { $each: ['Jon Arryn']}}});
```